packages/core/upload/admin/src/components/EditFolderDialog/EditFolderDialog.js

```diff
@@ -0,0 +1,215 @@
+    try {
+      await editFolder({
+        ...folder,
+        ...values,
+        parent: values.parent.value ?? null,
+      });
+
+      toggleNotification({
+        type: 'success',
+        message: formatMessage({
+          id: getTrad('modal.folder-notification-success'),
+          defaultMessage: 'Folder successfully created',
+        }),
+      });
+
+      onClose({ created: true });
+    } catch (err) {
+      const errors = getAPIInnerErrors(err, { getTrad });
+      const formikErrors = Object.entries(errors).reduce((acc, [key, error]) => {
+        acc[key] = error.defaultMessage;
+
+        return acc;
+      }, {});
+
+      if (!isEmpty(formikErrors)) {
+        setErrors(formikErrors);
+      }
+    }
+  };
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  return (
+    <ModalLayout onClose={handleClose} labelledBy="title">
+      <ModalHeader>
+        <Typography fontWeight="bold" textColor="neutral800" as="h2" id="title">
+          {formatMessage({
+            id: getTrad('modal.folder.create.title'),
+            defaultMessage: 'Add new folder',
+          })}
+        </Typography>
+      </ModalHeader>
+
+      <ModalBody>
+        <Formik
+          validationSchema={folderSchema}
+          validateOnChange={false}
+          onSubmit={handleSubmit}
+          initialValues={initialFormData}
+        >
+          {({ values, errors, handleChange, setFieldValue }) => (
+            <Form noValidate>
+              <Grid gap={4}>
+                {folder && (
+                  <GridItem xs={12} col={12}>
+                    <ContextInfo
+                      blocks={[
+                        {
+                          label: formatMessage({
+                            id: getTrad('modal.folder.create.elements'),
+                            defaultMessage: 'Elements',
+                          }),
+                          value: folder.children.length,
+                        },
+
+                        {
+                          label: formatMessage({
+                            id: getTrad('modal.folder.create.creation-date'),
+                            defaultMessage: 'Creation Date',
+                          }),
+                          value: folder.createdAt,
+                        },
+                      ]}
+                    />
+                  </GridItem>
+                )}
+
+                <GridItem xs={12} col={6}>
+                  <TextInput
+                    label={formatMessage({
+                      id: getTrad('form.input.label.folder-name'),
+                      defaultMessage: 'Name',
+                    })}
+                    name="name"
+                    value={values.name}
+                    error={errors.name}
+                    onChange={handleChange}
+                  />
+                </GridItem>
+
+                <GridItem xs={12} col={6}>
+                  <Stack spacing={1}>
+                    <FieldLabel>
```

Comment:
Can you add a for attribute please 😄

Reply:
Oh sure. Good catch!

Reply:
Fixed in b8f52b8d7f95bf596958ed4038070ba7e4580b66